README.md

```diff
@@ -86,6 +63,14 @@ $ deepstate-angr --num_workers 4 -output_test_dir out $DEEPSTATE/build/examples/
         └── f1d3ff8443297732862df21dc4e57262.pass
  ```
 
+## Usage
+
+DeepState is a static library, used to write test harnesses, accompanied by command-line _executors_ written in Python. At this time, the best documentation are the [examples](/examples) and our [paper](https://www.cefns.nau.edu/~adg326/bar18.pdf).
+
+## Contributing
+
+All accepted PRs are awarded bounties by Trail of Bits. Join the #deepstate channel on the [Empire Hacking Slack](https://empireslacking.herokuapp.com/) to discuss ongoing development and claim bounties. Check the `[good first issue](https://github.com/trailofbits/deepstate/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)` label for suggested contributions.
```

Comment:
Backticks outside of a link render the full markdown of the link in github-flavored markdown, so this will render wrong